src/DefaultEmissionManager.sol

```diff
@@ -38,6 +41,11 @@ contract DefaultEmissionManager is Ownable2StepUpgradeable, IDefaultEmissionMana
         _disableInitializers();
     }
 
+    function reinitialize() external reinitializer(2) {
```

Comment:
Access control is missed on reinitializer(2) so it can be front-runned, it is anyway not a big issue cause there are no input params. Nevertheless it is usually better to have control over the execution as it might reset the timestamp unless we want to reinitialize it during the proxy upgrade tx (recommended). Please confirm.

Reply:
good point, we will use `upgradeAndCall` on the ProxyAdmin instead of just `upgrade` 👍

Reply:
Thanks, it is much better this way due to we are upgrading constants so we should avoid any temporal situation with new constant and still old state variables. Thanks a lot.

Reply:
updated the test in [2b5c007](https://github.com/0xPolygon/pol-token/pull/58/commits/2b5c0075e85e2e311b2750c9076adf9c95ab3e00)